google/cloud/bigquery/schema.py

```diff
@@ -67,6 +67,15 @@ class SchemaField(object):
 
         policy_tags (Optional[PolicyTagList]): The policy tag list for the field.
 
+        precision (Optional[int]):
+            Precison (number of digits) of fields with NUMERIC type.
```

Comment:
(nit) Also for BIGNUMERIC, thus maybe reword to (BIG)NUMERIC?

Reply:
done